tests/unit/test.config.js

```diff
@@ -905,6 +905,24 @@ describe('config', () => {
         });
     });
 
+    it('parses package.json file correctly', () => {
+      return withTempDir(
+        (tmpDir) => {
+          const configFilePath = path.join(tmpDir.path(), 'package.json');
+          fs.writeFileSync(
+            configFilePath,
+            `{
+                "name": "dummyPackage.json",
```

Comment:
technically, a `version` is required to make this valid so I think you should add `"version": "1.0.0"`